script/Deploy.s.sol

```diff
@@ -4,9 +4,28 @@
 pragma solidity 0.8.20;
 
 import {Script} from "forge-std/Script.sol";
+import {console2} from "forge-std/console2.sol";
+import {CreatorTokenFactory} from "src/CreatorTokenFactory.sol";
+import {ITestableShowtimeVerifier} from "test/interfaces/ITestableShowtimeVerifier.sol";
 
 contract Deploy is Script {
-  function run() public {
+  CreatorTokenFactory creatorTokenFactory;
+  ITestableShowtimeVerifier verifier;
+  bytes32 domainSeparator;
+
+  /// @notice Deploy the contract
+  function run(address _verifierAddress) public {
```

Comment:
Remind me how this works? We have to pass this in as an argument?

Reply:
Just have to add in `--sig "run(address)" <ShowtimeVerifier Contract Address>` bit into the `forge script` command